scripts/yse/requirements.txt

```diff
@@ -0,0 +1,4 @@
+sncosmo==2.10.02
```

Comment:
should we just add these to the main requirements.txt file?

Reply:
The requirements.txt might be read in recursively anyways for usage?